public/app/features/plugins/admin/pages/Browse.test.tsx

```diff
@@ -39,46 +39,88 @@ function setup(path = '/plugins'): RenderResult {
 
 describe('Browse list of plugins', () => {
   it('should list installed plugins by default', async () => {
-    const { getByText, queryByText } = setup('/plugins');
+    const { queryByText } = setup('/plugins');
 
-    await waitFor(() => getByText('Installed'));
+    await waitFor(() => queryByText('Installed'));
 
     for (const plugin of installed) {
-      expect(getByText(plugin.name)).toBeInTheDocument();
+      expect(queryByText(plugin.name)).toBeInTheDocument();
     }
 
     for (const plugin of remote) {
       expect(queryByText(plugin.name)).toBeNull();
     }
   });
 
-  it('should list all plugins when filtering by all', async () => {
-    const plugins = [...installed, ...remote];
-    const { getByText } = setup('/plugins?filterBy=all');
+  it('should list all plugins (except core plugins) when filtering by all', async () => {
+    const { queryByText } = setup('/plugins?filterBy=all?filterByType=all');
```

Comment:
**Question:** as far as I understand if they just load `/plugins` then the result is going to be similar, except that it will then show the core plugins as well, right? Is that intended? Also, are we planning to be more explicit with these query URLs, or more like "omitting a filter = show all"?